plugins/Morpheus/stylesheets/uibase/_languageSelect.less

```
@@ -10,3 +13,25 @@
     transition: box-shadow 0s !important;
   }
 }
+@media screen and (max-width: 992px) {
```

The first part is not needed

```suggestion
@media (max-width: 992px) {
```

---

Done 😁